### PR TITLE
Enable HTTP/S toggling

### PR DESCRIPTION
**Pull request by mkdirJava**
## Summary
Enable HTTP/S toggle via application.properties

## Issue Type
- New Feature

## Starter Names
  - storage spring boot starter

## Additional Information
Small change, storage on azure is default to https

### CODE REVIEW

File: azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java [hunk 1]
```diff
@@ -50,15 +51,21 @@ public ServiceURL createServiceUrl(@Autowired(required = false) PipelineOptions
...
                 properties.getAccountKey());
-        final URL blobUrl = new URL(String.format(BLOB_URL, properties.getAccountName()));
+        final URL blobUrl = getURL();
         final PipelineOptions pipelineOptions = buildOptions(options);
         final ServiceURL serviceURL = new ServiceURL(blobUrl, StorageURL.createPipeline(credentials, pipelineOptions));
 
         return serviceURL;
     }
+    
+    private URL getURL() throws MalformedURLException{
```
[the remark this answers is not in the document]
Comment (mkdirJava) [reply azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java]: Shall I make changes and update this request? Or is it in?

[the remark this answers is not in the document]
Comment (Incarnation-p-lee) [reply azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java]: you can commit another change append this PR.

**Comment by mkdirJava** [comment]
@Incarnation-p-lee 
Spaces added

File: azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Incarnation-p-lee) [reply azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java]: Thanks a lot.

[the remark this answers is not in the document]
Comment (mkdirJava) [reply azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java]: Cheers Pan, First PR, hopfully I can contribute more. We are interested in developing Spring applications for Azure at work

**Pull request merged**